coderd/users_test.go

```diff
@@ -2273,6 +2346,28 @@ func onlyUsernames[U codersdk.User | database.User](users []U) []string {
 	return out
 }
 
+// dbUserToSDKUser converts database.User to codersdk.User
+func dbUserToSDKUser(dbUser database.User) codersdk.User {
```

Comment:
We fortunately already have this function, `db2sdk.User`. It takes a `database.User` and a slice of organization IDs. In this case that slice should just be `[]uuid.UUID{first.OrganizationID})`

Reply:
thanks for pointing this out!